receiver/pulsarreceiver/config_test.go

```diff
@@ -30,7 +30,6 @@ func TestLoadConfig(t *testing.T) {
 		Endpoint:              "pulsar://localhost:6500",
 		ConsumerName:          "otel-collector",
 		Subscription:          "otel-collector",
-		Encoding:              defaultEncoding,
```

Comment:
What is the default behavior if `encoding_extension` isn't set? I believe it should be OTLP encoding